### PR TITLE
EventGrid: 2019-06-01 .NET SDK generation

### PR DESCRIPTION
**Pull request by ahamad-MS**
Add the generated .NET SDK for 2019-06-01 API Version.

### CODE REVIEW

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please generate the code using generate.ps1 script and commit changes to the .txt log file too.

**Comment by ahamad-MS** [comment]
Thanks @dsgouda .. I added the ../../AzSdk.RP.props file.

**Comment by dsgouda** [comment]
@ahamad-MS I was referring to the .txt file that should have some changes when you generate the code using generate.ps1 script

**Comment by ahamad-MS** [comment]
@dsgouda I added the src\sdks\_metadata\eventgrid_resource-manager.txt file.

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please bump the version to `4.1.0-preview` in csproj/assemblyinfo.cs files and update the `PackageReleaseNotes`

**Comment by ahamad-MS** [comment]
@dsgouda Updated the release notes and assembly info with version 5.0.0 as this is GA version for us.

File: src/SDKs/EventGrid/management/Management.EventGrid/Microsoft.Azure.Management.EventGrid.csproj [hunk 1]
```diff
@@ -6,10 +6,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.EventGrid</PackageId>
     <Description>Provides developers with a library to create and manage all Azure EventGrid resources. </Description>
-    <Version>4.0.1-preview</Version>
+    <Version>5.0.0</Version>
```
Comment (dsgouda) [inline src/SDKs/EventGrid/management/Management.EventGrid/Microsoft.Azure.Management.EventGrid.csproj]: Are you sure you wish to publish a stable version right away, generally we recommend publishing a few preview versions before going stable, that way you have some leeway to implement fixes and such which wouldn't be a breaking change to customers.

Comment (ahamad-MS) [reply src/SDKs/EventGrid/management/Management.EventGrid/Microsoft.Azure.Management.EventGrid.csproj]: These features had been in preview for quite sometime. We had some of them introduced in 2018-09-15-preview and in 2019-02-01-preview. We are GA'ing them here in 2019-06-01. So, we are good to go.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**